SPTDataLoader/SPTDataLoaderRequestTaskHandler.m

```diff
@@ -108,7 +108,11 @@ - (void)receiveData:(NSData *)data
         if (self.request.chunks) {
             [self.requestResponseHandler receivedDataChunk:dataRange forResponse:self.response];
         } else {
-            [self.receivedData appendData:dataRange];
+            if (!self.receivedData) {
+                self.receivedData = [dataRange mutableCopy];
```

Comment:
Does this change may fix a latent bug where we assumed that `receivedData` was always non-nil? Do we have a test that exposes this bug?

Reply:
It only applies to `backgroundPolicy = .Always` which doesn’t receive an initial response (and thus wasn’t initializing `receivedData`). I added a test for this behavior.